doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -614,148 +619,101 @@ syntax: :n:`let fix @ident @binders := @term in` stands for
 
 .. _gallina-assumptions:
 
 Assumptions
 -----------
 
 Assumptions extend the environment with axioms, parameters, hypotheses
-or variables. An assumption binds an :token:`ident` to a :token:`type`. It is accepted
-by Coq if and only if this :token:`type` is a correct type in the environment
-preexisting the declaration and if :token:`ident` was not previously defined in
-the same module. This :token:`type` is considered to be the type (or
-specification, or statement) assumed by :token:`ident` and we say that :token:`ident`
-has type :token:`type`.
+or variables. An assumption binds an :n:`@ident` to a :n:`@type`. It is accepted
+by Coq if and only if this :n:`@type` is a correct type in the environment
+preexisting the declaration and if :n:`@ident` was not previously defined in
+the same module. This :n:`@type` is considered to be the type (or
+specification, or statement) assumed by :n:`@ident` and we say that :n:`@ident`
+has type :n:`@type`.
 
 .. _Axiom:
 
-.. cmd:: Parameter @ident : @type
-
-   This command links :token:`type` to the name :token:`ident` as its specification in
-   the global context. The fact asserted by :token:`type` is thus assumed as a
-   postulate.
-
-   .. exn:: @ident already exists.
-      :name: @ident already exists. (Axiom)
-      :undocumented:
-
-   .. cmdv:: Parameter {+ @ident } : @type
-
-      Adds several parameters with specification :token:`type`.
-
-   .. cmdv:: Parameter {+ ( {+ @ident } : @type ) }
-
-      Adds blocks of parameters with different specifications.
+.. cmd:: @assumption_token {? Inline {? ( @num ) } } {| {+ ( @assumpt ) } | @assumpt }
+   :name: Axiom; Axioms; Conjecture; Conjectures; Hypothesis; Hypotheses; Parameter; Parameters; Variable; Variables
 
-   .. cmdv:: Local Parameter {+ ( {+ @ident } : @type ) }
-      :name: Local Parameter
+   .. insertprodn assumption_token of_type
 
-      Such parameters are never made accessible through their unqualified name by
-      :cmd:`Import` and its variants. You have to explicitly give their fully
-      qualified name to refer to them.
+   .. prodn::
+      assumption_token ::= {| Axiom | Axioms }
+      | {| Conjecture | Conjectures }
+      | {| Parameter | Parameters }
+      | {| Hypothesis | Hypotheses }
+      | {| Variable | Variables }
+      assumpt ::= {+ @ident_decl } @of_type
+      ident_decl ::= @ident {? @univ_decl }
+      of_type ::= {| : | :> | :>> } @term
 
-   .. cmdv:: {? Local } Parameters {+ ( {+ @ident } : @type ) }
-             {? Local } Axiom {+ ( {+ @ident } : @type ) }
-             {? Local } Axioms {+ ( {+ @ident } : @type ) }
-             {? Local } Conjecture {+ ( {+ @ident } : @type ) }
-             {? Local } Conjectures {+ ( {+ @ident } : @type ) }
-      :name: Parameters; Axiom; Axioms; Conjecture; Conjectures
+   This command binds one or more :n:`@ident`\s to specified :n:`@type`\(s) as their specifications in
+   the global context. The fact asserted by the :n:`@type` is accepted as a
```

Comment:
```suggestion
   the global context. The fact asserted by the :n:`@type` / the existence of an object of this type is accepted as a
```

Reply:
That sort of makes sense, but is there a precise difference between asserting a fact and asserting the existence of an object?  Or is this saying the same thing in 2 different ways?  Can we give examples.  It's a bit fuzzy with the OR.